### PR TITLE
Refactored constants

### PR DESCRIPTION
**Pull request by Archonic944**
- No more hungarian notation
- Moved inner classes out of their coccoons (new package is frc.constants)

LGTM!

### CODE REVIEW

**Comment by Archonic944** [comment]
LGTM! 💯

**Comment by Archonic944** [comment]
Uhh... why haven't we auto-merged yet? 🤨

File: src/main/java/frc/constants/AutoConstants.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package frc.constants;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+public class AutoConstants {
+    public static final double K_MAX_SPEED_METERS_PER_SECOND = 3;
```
Comment (BlueZeeKing) [inline src/main/java/frc/constants/AutoConstants.java]: Forgot a K here

**Comment by Omerdad** [comment]
You left a K 🤓

File: src/main/java/frc/constants/ModuleConstants.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    public static final double DRIVING_ENCODER_VELOCITY_FACTOR = ((WHEEL_DIAMETER_METERS * Math.PI)
+            / DRIVING_MOTOR_REDUCTION) / 60.0; // meters per second
+
+    public static final double TURNING_ENCODER_POSITION_FACTOR = (2 * Math.PI); // radians
+    public static final double TURNING_ENCODER_VELOCITY_FACTOR = (2 * Math.PI) / 60.0; // radians per second
+
+    public static final double TURNING_ENCODER_POSITION_PID_MIN_INPUT = 0; // radians
+    public static final double TURNING_ENCODER_POSITION_PID_MAX_INPUT = TURNING_ENCODER_POSITION_FACTOR; // radians
+    /**
+     * A PID constant
```
Comment (BlueZeeKing) [inline src/main/java/frc/constants/ModuleConstants.java]: Put one comment for all not one for each

**Review by BlueZeeKing — CHANGES_REQUESTED** [review]
~~I think constants should be a subpackage of robot like the subsystems package, not a seperate one~~ nvm

**Comment by BlueZeeKing** [comment]
Also PID constants might make sense in it's own file

File: src/main/java/frc/constants/ModuleConstants.java (hunk 2 above)
Comment (BlueZeeKing) [reply src/main/java/frc/constants/ModuleConstants.java]: Also PID constants might make sense in it's own file

**Review by BlueZeeKing — CHANGES_REQUESTED** [review]
PID constants might make sense in it's own file

**Review by BlueZeeKing — DISMISSED** [review]
LGTM!

**Review by BlueZeeKing — APPROVED** [review]
Formatting is good enough

**Pull request merged**